container/crio/client.go

```diff
@@ -28,6 +29,11 @@ const (
 	maxUnixSocketPathSize = len(syscall.RawSockaddrUnix{}.Path)
 )
 
+var (
+	theClient      crioClient
+	clientErr      error
```

Comment:
clientErr is always nil.  No need to store it.

Reply:
@dashpole 
Can you take a look at #2460 as well ?
I intend to rebase this after #2460 goes in.

Reply:
I have rebased the PR.